src/Workspaces/Core/Portable/Storage/SQLite/SQLitePersistentStorage_BulkPopulateIds.cs

```diff
@@ -53,7 +53,7 @@ private void BulkPopulateProjectIds(SqlConnection connection, Project project, b
                 // the current string table, it will keep having problems trying to bulk populate.
                 if (fetchStringTable)
                 {
-                    if (!TryFetchStringTable(connection))
+                    if (!TryFetchStringTableFromMainDB(connection))
```

Comment:
note: all the string table work is done with the main DB.  this keeps us having a single consistent place for the string information.  It's also not a place we need to optimize as the string writing is already done in a bulk in a single transactin.